### ISSUE TITLE
Drink description missplaced

### ISSUE THREAD

**Issue by klaseskilson** [klaseskilson/vattentornet#108]
When a drink has an image, the drink description gets misplaced.

<img width="685" alt="screen shot 2016-02-23 at 12 53 58" src="https://cloud.githubusercontent.com/assets/1674076/13250845/ae4577e2-da2c-11e5-89cd-78fffda1c5be.png">

**Comment by tistatos** [comment]
You mean the description should be in the left side column?

**Comment by klaseskilson** [comment]
Yes, it used to be like that. The layout adapts to the image, and should be where it is now only when there is no image.

**Comment by tistatos** [comment]
Aah Okej
